htmlproofer/plugin.py

```diff
@@ -59,8 +68,54 @@ def get_url_status(self, url: str, soup: BeautifulSoup) -> Tuple[str, int]:
             except requests.exceptions.ConnectionError:
                 return clean_url, -1
         else:
+            match = re.match(r'(.+)#(.+)', clean_url)
+            if match is not None:
+                # URL is a link to another local Markdown file that includes an anchor.
+                url_target, anchor = match.groups()
+                target_markdown = self.find_source_markdown(url_target, files)
```

Comment:
are source and target markdown same here ?

Reply:
I clarified the method name to make these more consistent. Let me know if anything else needs clarified. I'm hoping the type annotations help with this as well.